swatch-contracts/src/test/java/com/redhat/swatch/contract/service/SubscriptionSyncServiceTest.java

```diff
@@ -339,7 +341,7 @@ void shouldSyncSubscriptionsSkipSubIfTooExpired() {
   void shouldSyncSubscriptionsSkipSubIfTooFutureDated() {
     // When syncing an org's subs, don't sync subscriptions future-dated more than 2 months from now
     var dto = createDto("456", 10);
-    dto.setEffectiveStartDate(toEpochMillis(NOW.plusMonths(2).plusDays(1)));
```

Comment:
Was it intentional to add `plusMinutes`?

Reply:
Yes, when troubleshooting this failure in my local using the timezone from Spain, apart from the hours, I spot a minor difference of a few seconds due to slow executions (my laptop is much slower than the old one ... ) , that's why adding the minutes will make this test more stable.